enterprise/internal/batches/reconciler/plan.go

```diff
@@ -106,26 +108,29 @@ func (p *Plan) SetOp(op btypes.ReconcilerOperation) { p.Ops = Operations{op} }
 // It consumes the current and the previous changeset spec, if they exist. If
 // the current ChangesetSpec is not applied to a batch change, it returns an
 // error.
-func DeterminePlan(previousSpec, currentSpec *btypes.ChangesetSpec, ch *btypes.Changeset) (*Plan, error) {
+func DeterminePlan(previousSpec, currentSpec *btypes.ChangesetSpec, currentChangeset *btypes.Changeset, wantedChangeset *btypes.Changeset) (*Plan, error) {
```

Comment:
No need to change this unless we have to crack this back open for something else, but you can elide the type if you have consecutive parameters of the same type; eg:

```suggestion
func DeterminePlan(previousSpec, currentSpec *btypes.ChangesetSpec, currentChangeset, wantedChangeset *btypes.Changeset) (*Plan, error) {
```

Reply:
Ahh, should have noticed `previousSpec` and `currentSpec`. I'm more of an explicit typing person 😀